src/cascadia/TerminalApp/App.cpp

```diff
@@ -251,23 +253,65 @@ namespace winrt::TerminalApp::implementation
         const auto package = winrt::Windows::ApplicationModel::Package::Current();
         const auto packageName = package.DisplayName();
         const auto version = package.Id().Version();
+        Windows::UI::Xaml::Media::SolidColorBrush blueBrush{ Windows::UI::ColorHelper::FromArgb(255, 0, 115, 207) };
+        winrt::Windows::UI::Xaml::Documents::Run about;
+        winrt::Windows::UI::Xaml::Documents::Run gettingStarted;
+        winrt::Windows::UI::Xaml::Documents::Run documentation;
+        winrt::Windows::UI::Xaml::Documents::Run releaseNotes;
+        winrt::Windows::UI::Xaml::Documents::Hyperlink gettingStartedLink;
+        winrt::Windows::UI::Xaml::Documents::Hyperlink documentationLink;
+        winrt::Windows::UI::Xaml::Documents::Hyperlink releaseNotesLink;
+        winrt::hstring gettingStartedLabel = L"Getting Started\n";
+        winrt::hstring documentationLabel = L"Documentation\n";
+        winrt::hstring releaseNotesLabel = L"Release Notes\n";
         std::wstringstream aboutTextStream;
 
+        gettingStarted.Text(gettingStartedLabel);
+        documentation.Text(documentationLabel);
+        releaseNotes.Text(releaseNotesLabel);
+
+        std::wstring gettingStartedUriValue{ L"https://github.com/microsoft/terminal/blob/master/doc/user-docs/UsingJsonSettings.md" };

```

Comment:
Should we make these aka.ms links that point at these files, in case we want to change the targets later?

Reply:
Absolutely. We could-should also do that for the Feedback link ;P